### PR TITLE
Support for 1.28 and aviinfrasetting nsxLr section add for v1alpha1

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

File: helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml [hunk 1]
```diff
@@ -103,6 +103,15 @@ spec:
...
                 - shardSize
+              nsxSettings:
+                properties:
+                  project:
+                    type: string
+                  t1lr:
+                    type: string
+                type: object
+                required:
+                - t1lr
```
Comment (abhishekbsingh) [inline helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml:114]: Don't we need to autogenerate code for this change?

**Review by abhishekbsingh — APPROVED** [review]

**Review by DixitAakash — DISMISSED** [review]
LGTM

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml (hunk 1 above)
Comment (akshayhavile) [reply helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml:114]: @abhishekbsingh: This field was already added. But ran that tool. It has created new set of definition for k8 functions as it downloaded new packages. So done this for v1beta1 and v1alpha2

**Review by abhishekbsingh — DISMISSED** [review]

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Review by arihantg — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

**Review by abhishekbsingh — APPROVED** [review]

**Pull request merged**